app.js

```diff
@@ -108,5 +120,76 @@ else{ //continue to quiz questions
       alert('You must answer YES or NO.');
     }
   }while(nInput(dog) === 'ERROR');
-  alert('That\'s all! Thank you for taking my quiz ' + userName + '!');
+
+  var correctAnswerSix = false; //this is ONLY to keep "I was actually thinking" prompt from running if correct answer is chosen
+  var randInt = Math.floor(Math.random() * 10) + 1; // setting this variable here so it doesn't get reset in the loop
+  for(var trySix = 0 ; trySix < 4; trySix++){ //question six
+    correctAnswerSix = false;
+    var numGuess = prompt('What number am I thinking?' , 'pick a whole between 1 and 10');
+    console.log('random Int = ' + randInt + ' user input = ' + numGuess + ' is an integer ' + Number.isInteger(parseInt(numGuess)));
+
+    if(isNaN(parseInt(numGuess)) || !Number.isInteger(parseInt(numGuess)) || numGuess > 10){ //catch entries that won't evaluate
+      alert('You need to guess a number between 1 and 10');
+    }
+    else if(numGuess == randInt){
+      correctAnswerSix = true;
+      alert('WOW, great guess ' + userName + '! My number was' + randInt);
+      numCorrect++;
+      console.log('numCorrect set to ' + numCorrect);
+      break;
+    }
+    else{
+      alert('Nope, not ' + numGuess + '.');
+    }
+  }
+  if (correctAnswerSix === false){
+    alert('I was actually thinking ' + randInt + '!');
+  }
+
+  var correctAnswerSevens = ['Iowa','Wisconsin','Louisiana'];
+  var validAnswerSevens = ['IOWA','IA','WISCONSIN','WI','LOUISIANA','LA'];
+  var correctAnswerSeven = false;
```

Comment:
In the future I might pay closer attention to variable names. When working it might be hard to tell the difference, or easy for someone to miss when working in your code, that correctAnswerSevens and correctAnswerSeven are different vars.